src/test/java/com/commercetools/project/sync/state/StateSyncerTest.java

```diff
@@ -44,15 +43,15 @@ void transform_ShouldReplaceStateTransitionIdsWithKeys() {
             readObjectFromResource("state-2.json", State.class));
 
     final CompletionStage<List<StateDraft>> stateDrafts = stateSyncer.transform(states);
-    final StateDraftDsl draft1 =
+    final StateDraft draft1 =
```

Comment:
can you add `//preparation //test //assertions` sections into this test case to be more consistent?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/commits/752bc763cd677d5091303f5b83e3b84148f6ec41